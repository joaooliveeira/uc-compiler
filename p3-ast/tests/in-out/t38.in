int var = (1 < 4.0) && (5 >= 10) || (y != x);
