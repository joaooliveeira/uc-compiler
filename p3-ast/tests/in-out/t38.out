Program:
    GlobalDecl:
        Decl: ID(name=var)
            VarDecl:
                Type: int @ 1:1
            BinaryOp: || @ 1:12
                BinaryOp: && @ 1:12
                    BinaryOp: < @ 1:12
                        Constant: int, 1 @ 1:12
                        Constant: float, 4.0 @ 1:16
                    BinaryOp: >= @ 1:25
                        Constant: int, 5 @ 1:25
                        Constant: int, 10 @ 1:30
                BinaryOp: != @ 1:38
                    ID: y @ 1:38
                    ID: x @ 1:43
